{
    "uvi": [
        {
            "extracted_data": {
                "advisory_id": "DSA-666-1",
                "advisory_type": "DSA",
                "affected": [
                    {
                        "distribution_affected": "woody",
                        "fixed in": "2.2.1-4.7. No other version of Python in woody is affected"
                    },
                    {
                        "distribution_affected": "sarge) and unstable (sid",
                        "fixed in": "For the testing (sarge) and unstable (sid) distributions the following matrix explains which version will contain the correction in which version:"
                    }
                ],
                "aliases": [
                    {
                        "ids": [
                            "CVE-2005-0089"
                        ],
                        "namespace": "cve.mitre.org"
                    },
                    {
                        "ids": "DSA-666-1",
                        "namespace": "www.debian.org"
                    }
                ],
                "info_date_string": "2005-02-04",
                "package_name": "python2.2",
                "product_name": "Linux",
                "vendor_name": "Debian",
                "vulnerability_status": "yes"
            },
            "meta_data": {
                "uvi_advisory_formatting_issues": "no",
                "uvi_processed_timestamp": "2021-08-03T03:18:13.854298Z",
                "uvi_script_hash": "181554e37567de9a4b4069b78e473a8edce1bc5e80c4f65bd36734d2b48cf601618b527163f2371322d7d2c974fcc72e50a6617b535c5a38df7add95cb74dcdf",
                "uvi_script_name": "./www.debian.org-processing.py",
                "uvi_script_version": "0.1.2",
                "uvi_url_processed": "https://www.debian.org/security/2005/dsa-666"
            }
        }
    ]
}